client/my-sites/comments/comment-list/index.jsx

```diff
@@ -566,6 +569,18 @@ const mapDispatchToProps = ( dispatch, { siteId } ) => ( {
 			)
 		),
 
+	recordBulkAction: ( action, count, fromList ) =>
+		dispatch(
+			composeAnalytics(
+				recordTracksEvent( 'calypso_comment_management_bulk_action', {
+					action,
+					count,
+					from_list: fromList,
```

Comment:
I wasn't sure what values would be recorded here just from the name. I think it would be more clear if we used `from_status`, and `fromStatus` above instead of `fromList` and `listStatus`.

Since we are recording this we could also add `to_status`.

Reply:
👍  I think I like `from_status` over `from_list`. We'll be adding in a post/comment specific view in soon. Would it be helpful to also add the site/post/comment filter as well?

Reply:
For the single actions which still fire, we list the previous status, instead of it's position. With the 'all' filter, should we also consolidate that to use `from_status` over `previous_status`?

Reply:
The thing is, `status` could be a bit misleading because it can very well happen that all selected comments don't have the same `previousStatus`.
Think of the All, Pending, and Approved lists: all of them can contain both `unapproved` and `approved` comments, which can be selected together.
I'm not sure how to handle these cases.

I chose the term `list` because it indicates where the user is when they do a bulk action, while each single change is tracked by the single action analytics.

Also, replacing it to `status` would mean for example that we wouldn't know about actions fired from the All list anymore.

Reply:
We are using `all` and `pending` under `status` umbrella in other places too: https://github.com/Automattic/wp-calypso/pull/19007/files#diff-2337249ec06f61991a2ed13206f9c520R11. As long as we are consistent with naming things it's fine with me. :)

> Also, replacing it to status would mean for example that we wouldn't know about actions fired from the All list anymore.

I wasn't suggesting to replace the actual values being recorded, just the variable names in the code for easier readability.

Reply:
Let's go ahead with this and be sure to describe what `from_list` does in the tracks event description. Feel free to ping me in Slack if you had questions on how to do that.

Reply:
If we're tracking `from_list` a view param might be good here too, eg `view: site/post/comment`

Reply:
@vindl Yeah, my explanation was unclear. :)

> We are using all and pending under status umbrella in other places too

True, but we don't use `all` as a "comment status", just as a "list status", if you get what I mean.
To reword in a natural language, when we track the event we'd say "this comment changed from `pending` to `approved`", not "this comment changed from `all` to `approved`".

PS: I've got no grudge against `pending`. :)